index.html

```diff
@@ -169,13 +169,32 @@ <h3 class="lang-text">CSS</h3>
           <div class="lang lang-2">
             <h3 class="lang-font">Frameworks</h3>
             <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
-          </div>
+       </div>
           <div class="lang lang-2">
             <h3 class="lang-font">Skills</h3>
             <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
           </div>
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-form">
+        <div class="contact-heading">
+          <h2 class="con-title">Contact me</h2>
+          <p class="con-paragraph">
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
```

Comment:
- [ ] You did an amazing job on this milestone but please move the text more to the left following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159912997-eeabacba-67d4-4aeb-9f1c-8e9c87648679.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159913040-41bcd98a-06e5-4a07-b858-f7eef70f843d.png)